### ISSUE TITLE
error: could not create compact unwind for _blst_sha256_block_data_order: does not use RBP or RSP based frame

### ISSUE THREAD

**Issue by xerosanyam** [flashbots/mev-boost#251]
unable to install mev-boost
`go install github.com/flashbots/mev-boost@latest`
results in error: `could not create compact unwind for _blst_sha256_block_data_order: does not use RBP or RSP based frame`

System Information:
Mac OS
go version go1.19 darwin/amd64

**Comment by metachris** [comment]
is this an error or a warning? could you post the full logs? it's related to the BLST library.

**Comment by metachris** [comment]
Should be fixed with #258 


```
CGO_CFLAGS=-"O -D__BLST_PORTABLE__" go install github.com/flashbots/mev-boost@latest
```
